### PR TITLE
Mobile testing enablement adding Appium & Sauce Labs in e2e-spock-geb quickstarter

### PR DESCRIPTION
**Pull request by garcanam**
Mobile testing enablement
Incorporating mobile testing technology with Appium and SauceLab into our current Spock&Geb QS.

### CODE REVIEW

File: e2e-spock-geb/files/README.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+def sauceLabsAccessKey = System.getenv('SAUCE_LABS_ACCESS_KEY')
+...
+ios {
+driver = {
+...
+IOSDriver driver = new IOSDriver(new URL("https://$sauceLabsUsername:$sauceLabsAccessKey@ondemand.eu-central-1.saucelabs.com:443/wd/hub"), caps);
+return driver
+}
+```
+
```
Comment (metmajer) [inline e2e-spock-geb/files/README.md:61]: @garcanam if users decide to use SauceLabs, where would they put this? I believe that some more guidance would be beneficial.

File: e2e-spock-geb/files/README.md [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+### Environments
+
+This section defines different environments such as DESKTOP, MOBILE_BROWSER, and MOBILE_APP. Each environment has its own driver setup.
```
Comment (metmajer) [inline e2e-spock-geb/files/README.md:64]: @garcanam same question as above. Can we add a little bit of information to guide our users about where to put such configurations?

Comment (garcanam) [reply e2e-spock-geb/files/README.md:48]: @metmajer I added more info in README and *.adoc.

**Review by metmajer — APPROVED** [review]
Good job @garcanam, I congratulate you on the clean code, especially how you shifted complexity/logic into build.gradle.

**Pull request merged**